### DISCUSSION TITLE
Execution of hammerdbcli fails

### DISCUSSION THREAD

**Discussion by mamiyaakira** [TPC-Council/HammerDB#346] [Q&A] [answered]
You are running hammerdbcli, but if you run buildschema,
The following error is output:.
Please tell me how to deal with it.

Error in Virtual User 1: invalid command name "pg_notice_handler"

**Comment by sm-shaw** [comment, marked as answer]
The package Pgtcl has not been loaded, an example is shown that this command is part of the Pgtcl package.
```
hammerdb>pg_notice_handler
Error: invalid command name "pg_notice_handler"

hammerdb>package require Pgtcl
2.1.1
hammerdb>pg_notice_handler
wrong # args: should be "connection ?command?"
    while executing
"pg_notice_handler "
hammerdb>
```
The loading of the package is the first thing that buildschema does. Look for the "Failed to load" message earlier.
```
#!/usr/local/bin/tclsh8.6
#LOAD LIBRARIES AND MODULES
set library Pgtcl
if [catch {package require $library} message] { error "Failed to load $library - $message" }
```
Pgtcl is in the lib directory. If the package cannot be found, it is possible your installation is corrupted.

**Reply by mamiyaakira** [reply]
The path to pgtcl did not seem to be correct. It worked properly after setting. Thank you very much.
